src/core-sdk-tasks/ExtractArchiveToDirectory.cs

```diff
@@ -73,13 +78,41 @@ public override bool Execute()
             {
                 if (ValidateParameters())
                 {
+                    if (DirectoriesToCopy != null && DirectoriesToCopy.Length != 0)
+                    {
+                        var zip = new ZipArchive(File.OpenRead(SourceArchive));
+                        string loc = DestinationDirectory;
+                        foreach (var entry in zip.Entries)
+                        {
+                            foreach (var directory in DirectoriesToCopy.Split(';'))
+                            {
+                                if (entry.FullName.Contains(directory))
+                                {
+                                    string dirBuilder = loc;
+                                    foreach (var pathPart in Path.GetDirectoryName(entry.FullName).Split('/'))
```

Comment:
Are zip entries always delimited by '/' even on Windows?

Reply:
When I was doing this work I had the full names of the paths inside the zip archive printed out, and this is how they were delimited. These were on ostensibly Windows zips.